### PR TITLE
Added the the get gain command

### PR DESCRIPTION
**Pull request by Sarthak031**

### CODE REVIEW

File: src/board_controller/brainalive/brainalive.cpp [hunk 1]
```diff
@@ -190,6 +192,7 @@ int BrainAlive::prepare_session ()
     if ((res == (int)BrainFlowExitCodes::STATUS_OK) && (control_characteristics_found))
     {
         initialized = true;
+        config_board ("0a036007000d");
```
Comment (Andrey1994) [inline src/board_controller/brainalive/brainalive.cpp:195]: Add check for result here

File: src/board_controller/brainalive/brainalive.cpp [hunk 2]
```diff
@@ -302,15 +305,20 @@ int BrainAlive::config_board (std::string config)
...
     }
-    uint8_t command[5];
-    command[0] = 0x0a;
-    command[1] = 0x81; // it is hardcoded for now only
-    command[2] = 0x00;
-    command[3] = 0x00;
-    command[4] = 0x0d;
-    safe_logger (spdlog::level::trace, config[2]);
+    // Calculate the number of bytes
+    size_t numBytes = config.length () / 2; // Each byte is represented by 2 hex characters
```
Comment (Andrey1994) [inline src/board_controller/brainalive/brainalive.cpp:309]: There is snake case with _ for variables everywhere in brainflow, lets keep it consistent

Comment (Andrey1994) [reply src/board_controller/brainalive/brainalive.cpp:309]: Applies to all variables in this block

**Comment by Andrey1994** [comment]
I also dont get why do you need to config board multiple times and what these configs mean, but that board speciffic part is up to your device to determine

File: src/board_controller/brainalive/brainalive.cpp (hunk 1 above)
Comment (Andrey1994) [reply src/board_controller/brainalive/brainalive.cpp:195]: So it should be res =... and below that initialized = true

Comment (Sarthak031) [reply src/board_controller/brainalive/brainalive.cpp]: done

File: src/board_controller/brainalive/brainalive.cpp (hunk 2 above)
Comment (Sarthak031) [reply src/board_controller/brainalive/brainalive.cpp]: done

**Comment by Sarthak031** [comment]
> I also dont get why you need to config board multiple times and what these configs mean, but that board speciffic part is up to your device to determine

The first config is for start data streaming, the second config is used for stop data streaming, and the third is used to get the configuration of the device before data streaming.

**Review by Andrey1994 — APPROVED** [review]

**Pull request merged**